### ISSUE TITLE
Swift package manager support

### ISSUE THREAD

**Issue by AndrewSB** [square/Valet#89]
Would be awesome to use Valet with swift package manager!

It's waiting on Swift 3 support, so it's probably something to implement after v3.0, and after support for Swift 3 is fleshed out? Not sure if that's targeted as part of v3.0

**Comment by dfed** [comment]
It would be awesome! We'll add this to our list of things to do with our Swift 3.0 work

**Comment by dfed** [comment]
I've done a first pass of this on #80. @AndrewSB would love you to take a look since I don't have much background with SPM.

Please note that we're keeping the old VAL*.h/m files around in Valet 3, but they'll only exist in the Test target (so that we can verify that our upgrade path is entirely seamless).

**Comment by dfed** [comment]
This is fixed with version 3.0!

**Comment by LinusU** [comment]
@dfed Are you sure that it is? I can't see any `Package.swift` file, and trying to use it gives the following error:

```text
Updating https://github.com/Square/Valet
error: noManifest(baseURL: "https://github.com/Square/Valet", version: Optional("3.0.0"))
```
